.prettierrc

```diff
@@ -1,5 +1,6 @@
 {
   "singleQuote": true,
   "semi": true,
-  "trailingComma": "all"
+  "trailingComma": "all",
+  "printWidth": 150
```

Comment:
@xkill-9, let's get this back to 80.

Per Prettier's website:

> For readability we recommend against using more than 80 characters:

![Options_·_Prettier_-_Firefox_Developer_Edition](https://user-images.githubusercontent.com/464978/72715056-cb6a3300-3b3d-11ea-9b31-de220e593e41.jpg)

Reply:
With that printWidth Prettier adds a bunch of `{ ' ' }` blocks as text breaks in long texts inside JSX and this breaks the translation strings used by `next-i18next`, i could add a bunch of `// prettier-ignore` comments on every long text line if you prefer.

Reply:
Actually that doesn't matter. Per `i18next` documentation translations don't depend on line length and it uses the element's index in the `children` array, for example:

```jsx
// line width: 80
<Trans i18nKey="how_to_get_to_the_conference_description">
  Medellín offers a metro system that will take you to the{' '}
  <Accent as="span">Universidad Station</Accent> that is just at 5
  minute walk to the Venue in around 10 minutes from{' '}
  <Accent as="span">Poblado Station</Accent>
</Trans>
```

Will be translated as:

![i18n](https://user-images.githubusercontent.com/464978/72980123-e809a380-3da7-11ea-9acb-62a1d955edcb.jpg)

Where `2` and `5` are other react elements that can be replaced with i18n strings by doing:

```json
"how_to_get_to_the_conference_description": "Medellín offers a metro system that will take you to the <2>Universidad Station</2> that is just at 5 minute walk to the Venue <5>in around 10 minutes from Poblado Station</5>",
```

Reply:
That said, let's set line width to 80.